.template-helpers/variant.sh

```diff
@@ -23,9 +24,16 @@ for tag in "${tags[@]}"; do
 	fi
 done
 if [ "$text" ]; then
-	# give a little space
-	echo
-	echo
-	cat "$dir/variant.md"
-	echo "$text"
+  latest=($(curl -sSL $url | sed -e 's/git:\/\/github.com\///' -e 's/@/ /' - | grep "latest"))
```

Comment:
Same comment here (`curl -sSL "$url" ...`) -- also, that `\/\/` looks hairy.  How about something like `sed -e 's!git://github.com/!!' ...` instead?